### PR TITLE
fix circular reference

### PR DESCRIPTION
**Pull request by rtio**

### CODE REVIEW

File: config/packages/easy_admin/entities/job_offer.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+easy_admin:
+    entities:
+        JobOffer:
+            class: App\Entity\JobOffer
+            list:
+                fields: ['id', 'title', 'company']
+            form:
+                fields:
+                    - { type: 'group', label: 'Delivery Details', icon: 'paperclip' }
```
Comment (ianrodrigues) [inline config/packages/easy_admin/entities/job_offer.yml]: Copy and paste detected. Should be "Job Offer Information".

Comment (ianrodrigues) [reply config/packages/easy_admin/entities/job_offer.yml]: Or "Job Offer Details", whatever keeping consistency around the application.

File: src/Controller/Api/JobOfferController.php [hunk 2]
```diff
@@ -25,20 +26,27 @@ public function __construct(JobOfferRepository $repository)
...
-        return $this->json($jobOffers);
+
+        return $this->json($jobOffers, Response::HTTP_OK, [], ['groups' => ['list']]);
     }
 
     /**
      * @Route("/api/job-offer/{jobOfferId}", name="api_job_offer_display")
      */
     public function display(int $jobOfferId): Response
     {
```
Comment (ianrodrigues) [inline src/Controller/Api/JobOfferController.php]: Empty line

File: config/packages/easy_admin/entities/job_offer.yml (hunk 1 above)
Comment (rtio) [reply config/packages/easy_admin/entities/job_offer.yml]: got me. will fix

File: src/Controller/Api/JobOfferController.php (hunk 2 above)
Comment (rtio) [reply src/Controller/Api/JobOfferController.php]: done

**Comment by rtio** [comment]
> My only concern is to change files out of the scope to the PR.

I agree. I'm going to consider it in next PRs.

**Pull request merged**